src/esp/metadata/attributes/LightLayoutAttributes.cpp

```diff
@@ -17,8 +17,8 @@ LightInstanceAttributes::LightInstanceAttributes(const std::string& handle)
   setIntensity(1.0);
   setType("point");
   // ignored for all but spot lights
-  setInnerConeAngle(0.75_radf);
-  setOuterConeAngle(1.5_radf);
+  setInnerConeAngle(0.0_radf);
+  setOuterConeAngle(1.57_radf);
```

Comment:
I mean, why not in degrees? `90.0_degf` "just works" ;)

Reply:
Maybe it is not related to this PR.
How can we adjust the lights during the runtime (e.g., in the viewer) currently? (what APIs should I call in the viewer, e.g., if I would like to increase the light intensity, or change colors.)

Reply:
> Maybe it is not related to this PR.
> How can we adjust the lights during the runtime (e.g., in the viewer) currently? (what APIs should I call in the viewer, e.g., if I would like to increase the light intensity, or change colors.)

Yes that is outside the scope of this PR - no actual lights are made from this code yet :).  All of these initial MM PRs are focused on the file io framework for parsing configuration files into attributes suitable for asset creation and subsequent modification.  The actual creation/modification code is in the future.